applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1065,6 +1122,7 @@ static int initialize(le_audio_receive_cb recv_cb)
 	for (int i = 0; i < ARRAY_SIZE(headsets); i++) {
 		headsets[i].sink_stream.ops = &stream_ops;
 		headsets[i].source_stream.ops = &stream_ops;
+		k_work_init_delayable(&headsets[i].stream_start_work, work_stream_start);
```

Comment:
Are the CIS's connected in the initialize function?

Reply:
No, this is just initializing the k_works